include/wx/textctrl.h

```diff
@@ -757,6 +757,8 @@ class WXDLLIMPEXP_CORE wxTextCtrlBase : public wxControl,
     // more readable flag testing methods
     bool IsSingleLine() const { return !HasFlag(wxTE_MULTILINE); }
     bool IsMultiLine() const { return !IsSingleLine(); }
+    bool IsMaxLengthAllowed() const { return m_maxLengthAllowed; }
```

Comment:
What is this function? If it's public, it must be documented. If not, it shouldn't be declared here, should have a `WX` prefix and still should be documented in a comment, at least.